deployment/CP05MOAS-GL335_Deploy.csv

```diff
@@ -1,31 +1,31 @@
-AR-04,,TS20160627,,CP05MOAS-GL335-03-CTDGVM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-CTDGVM-09089,39.83182,-70.72487,,200,200,FZ-2
-AR-04,,TS20160627,,CP05MOAS-GL335-04-DOSTAM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-DOSTAM-00171,39.83182,-70.72487,,200,200,FZ-2
-AR-04,,TS20160627,,CP05MOAS-GL335-05-PARADM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-PARADM-50167,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-00-ENG000000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGCON-GLDRCP-00335,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-01-ADCPAM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-ADCPAM-50867,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-02-FLORTM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-FLORTM-03207,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-03-CTDGVM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-CTDGVM-09089,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-04-DOSTAM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-DOSTAM-00171,39.83182,-70.72487,,200,200,FZ-2
+AR4,,TS20160627,,CP05MOAS-GL335-05-PARADM000,4,1,2016-05-27T20:33:00,2016-06-28T19:54:55,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-PARADM-50167,39.83182,-70.72487,,200,200,FZ-2
 WJ20170115,,TS20170306,,CP05MOAS-GL335-00-ENG000000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGCON-GLDRCP-00335,40.042333,-70.37853,,200,200,EB
 WJ20170115,,TS20170306,,CP05MOAS-GL335-01-ADCPAM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-ADCPAM-43106,40.042333,-70.37853,,200,200,EB
 WJ20170115,,TS20170306,,CP05MOAS-GL335-02-FLORTM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-FLORTM-04466,40.042333,-70.37853,,200,200,EB
 WJ20170115,,TS20170306,,CP05MOAS-GL335-03-CTDGVM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-CTDGVM-09364,40.042333,-70.37853,,200,200,EB
 WJ20170115,,TS20170306,,CP05MOAS-GL335-04-DOSTAM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-DOSTAM-00310,40.042333,-70.37853,,200,200,EB
-WJ20170115,,TS20170306,,CP05MOAS-GL335-05-PARADM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-PARADM-50237,40.042333,-70.37853,,200,200,EB
\ No newline at end of file
+WJ20170115,,TS20170306,,CP05MOAS-GL335-05-PARADM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-PARADM-50237,40.042333,-70.37853,,200,200,EB

```

Comment:
CUID updates look good.  Not quite sure about last line where it looks like it went from no new line at end of file to maybe a new line?